src/plugins/data_source_management/public/components/data_source_selector/data_source_selector.test.tsx

```diff
@@ -5,11 +5,16 @@
 
 import { ShallowWrapper, shallow } from 'enzyme';
 import { DataSourceSelector } from './data_source_selector';
-import { SavedObjectsClientContract } from '../../../../../core/public';
+import { SavedObjectsClientContract, IUiSettingsClient } from '../../../../../core/public';
 import { notificationServiceMock } from '../../../../../core/public/mocks';
 import React from 'react';
-import { getDataSourcesWithFieldsResponse, mockResponseForSavedObjectsCalls } from '../../mocks';
+import {
+  getDataSourcesWithFieldsResponse,
+  mockManagementPlugin,
+  mockResponseForSavedObjectsCalls,
+} from '../../mocks';
 import { AuthType } from 'src/plugins/data_source/common/data_sources';
+import * as utils from '../utils';
```

Comment:
Nit:
Can we try to avoid importing * if possible?

Thanks~

Reply:
Hi Xinrui. Thanks for bring this up. I am following the same practice of create_data_source_wizard.test.tsx. Spyon takes two arguments with object and method. I guess this is the reason why we import * as utils